ngi_reports/reports/ont_project_summary.py

```diff
@@ -0,0 +1,148 @@
+"""Module for producing the ONT Project Summary Report"""
+
+import os
+import ngi_reports.reports.project_summary
+
+from collections import OrderedDict
+from string import ascii_uppercase as alphabets
+
+
+class Report(ngi_reports.reports.project_summary.Report):
```

Comment:
Do you think we need to worry about namespace collisions?

Reply:
I think it's OK in this case. I agree that it's not optimal, but it's to make it work with this: https://github.com/NationalGenomicsInfrastructure/ngi_reports/blob/d3139444afdafe62b12f54fd1dbc1267a2058268/ngi_reports/ngi_reports.py#L66